mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -41,7 +59,7 @@ def ocn_modelvsobs(config, field):
 
     #Seems like the following line should be a config.get option and not
     #read every time series file when only a subset is taken
-    infiles = "".join([indir,"/am.mpas-o.timeSeriesStats.????-*.nc"])
+    #infiles = "".join([indir,"/am.mpas-o.timeSeriesStats.????-*.nc"])
```

Comment:
Thanks @xylar, this is great.  I'm liking what I'm seeing so far.  The only substantive comment following this discussion is related to pre-filtering timesteps via custom parsing when xarray should be able to do this natively.  I don't think this is needed yet (or ever) but if I'm missing something please correct me.